cfgov/templates/welcome.html

```diff
@@ -3,14 +3,42 @@
 
 {% block furniture %}
     <div class="content-wrapper">
+
         <h1>{% trans 'Welcome' %} <br/>
-        You've logged in to the consumerfinance.gov editorial tools.	
+        {% if 'build' in request.site.hostname %}
+            You've logged in to the demo Wagtail.    
+        {% else %}
+            You've logged in to the consumerfinance.gov editorial tools.    
+        {% endif %}
         </h1>
-        <h2>Continue on to...</h2>
+
+        <h2>Choose which admin interface you'd like to go to</h2>
         <ul>
-        {% for name, url in destinations %}
-            <li><a href="{{url}}"> {{name}}</a></li>
-        {% endfor %}
+            {% for name, url in destinations %}
+                <li><a href="{{url}}"> {{name}}</a></li>
+            {% endfor %}
         </ul>
+
+        {% if 'build' in request.site.hostname %}
+            <h2>Helpful links</h2>
+            <ul>
+                <li>
+                    <a href="https://github.cfpb.gov/pages/flapjack/wagtail-documentation/">
```

Comment:
@hillaryj While Adam is digging into the GHE linking issue, for this PR, please remove the link here, replace with a line of text and keep the design manual links. In total:

Helpful links:
- Design manual - Global elements
- Design Manual - Page components
- CFPB-specific Wagtail documentation (please see link in CF.gov Platform team repo under "playground")

I realize this isn't optimal, but it will help us move this PR through for now. Thanks all for the discussion!


Design Manual - Global elements: https://cfpb.github.io/design-manual/global-elements/global-elements.html
Design Manual - Page components: https://cfpb.github.io/design-manual/page-components/page-components.html